pkg/apis/eventing/v1alpha1/cluster_provisioner_types.go

```diff
@@ -66,7 +66,7 @@ type ClusterProvisionerSpec struct {
 	Reconciles metav1.GroupKind `json:"reconciles"`
 }
 
-var cProvCondSet = duckv1alpha1.NewLivingConditionSet()
+var cProvCondSet = duckv1alpha1.NewLivingConditionSet(ClusterProvisionerConditionProvisionerReady)
```

Comment:
I am not sure what `provisioner.status.condion.ready` vs `provisioner.status.condition.provisionerready` means. I suspect ProvsionerReady is redundant.  I will have to keep reading.